docs/proposals/2019-10-22-Geo-Restriction-Ids.md

```diff
@@ -0,0 +1,38 @@
+# Adding Geo-Restriction IDs to the content_restrictions Trait
+
+# Problem
+
+Certain clients need the ability to restrict content based on a user's geographical location, and to decide the amount of time that a piece of content is allowed to be publically viewed. To support this, documents must be associated with restrictions in some way.
+
+# Proposal
+
+We propose extending the `content_restrictions` trait to include a `geo` field, which maps to an object that contains the restriction information the document should be associated with. For the moment, only videos should be allowed to use the `geo` field.
+
+## restrictions
+
+The `geo` object should have a `restrictions` field. This field would map to an array of restriction objects, each of which contains a `restriction_id`. The `restrictions` array should be limited to a size of 1 for now.
+
+## Example
+```
+{
+  "type": "video",
+  "version": "0.10.4",
+  "content_restrictions": {
+    "geo": {
+      "restrictions": [{
+        "restriction_id": "abcdefghijklmnopqrstuvwxyz"
```

Comment:
Just curious what's the real geo restriction object should look like?

Reply:
The real object would look like that, for now, with just a `restriction_id` field. It's an object so that we can leave it open for adding other fields in the future, if necessary.

Reply:
I think @chenghsinw means the restriction definition itself, which we aren't defining here (this is just associating a document with a restriction defined elsewhere).

Reply:
Are `restriction_id`s arbitrary?

Reply:
Ok, then yeah, if this ID is a reference to another object, should that not be defined here?

Reply:
From what I understand, this reference won't be denormalized at any point, so it won't be part of the ANS document and doesn't need to be defined here.

Reply:
@gengel correct me if I'm wrong, or if that's not quite the right justification.

Reply:
From the ANS perspective, the ids are an arbitrary string, similar to distributor ids.

The Geo-Restrictions have their own API and spec -- that itself is not ready yet, but is somewhat outside the scope of this PR.

Reply:
@chenghsinw I think we need to figure it out in this ticket https://arcpublishing.atlassian.net/browse/AH-826